npe2/_version.py

```diff
@@ -1,4 +1,5 @@
+# coding: utf-8
```

Comment:
this file should actually be added to .gitignore ... I can give this repo a general pre-commit cleanup and do that

Reply:
ok, I just cleaned it all up quite a bit in #4  ... can you merge/rebase at some point?